onnxmltools/convert/sklearn/operator_converters/PCA.py

```diff
@@ -0,0 +1,24 @@
+# -------------------------------------------------------------------------
```

Comment:
Could you merge this into the converter of scikit-learn SVD? They are almost identical.